ddtrace/internal/remoteconfig/client.py

```diff
@@ -239,14 +239,14 @@ def base64_to_struct(val, cls):
         self.converter.register_structure_hook(SignedRoot, base64_to_struct)
         self.converter.register_structure_hook(SignedTargets, base64_to_struct)
 
-        self._products = dict()  # type: MutableMapping[str, ProductCallback]
+        self._products = dict()  # type: MutableMapping[str, Optional[ProductCallback]]
```

Comment:
Does this mean that keys in this dictionary can point to `None` values? It's not clear to me how the changes in this file are related to the rest of the diff.

Reply:
None, vestigial from the previous version of the base branch, removed.